Makefile

```diff
@@ -451,3 +451,8 @@ package-rpm-suse package-windows package-standalone-macosx \
 package-generic-unix: $(PACKAGES_SOURCE_DIST_FILE)
 	$(verbose) $(MAKE) -C packaging $@ \
 		SOURCE_DIST_FILE=$(abspath $(PACKAGES_SOURCE_DIST_FILE))
+
+generate-config:
+	rm -rf generated/rabbitmq.*.config
+	./cuttlefish -e . -s ./schema -c rabbitmq.conf.d/rabbitmq.conf -f rabbitmq
```

Comment:
I need to test it to make sure but downloading/compiling something is a matter of adding it to BUILD_DEPS and setting the appropriate dependent targets. I'm not sure if cuttlefish works well with Erlang.mk though (I know it compiles, I don't know if it produces a working escript). I'll experiment.